internal/version/version.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package version
+
+const (
+	// weaver-kube module version (Major.Minor.Patch).
+	Major = 0
+	Minor = 18
```

Comment:
Does this means that you release 0.18.0 for kube already?

Reply:
Nope, it just means that the next tagged release of weaver-kube will be 0.18.0. I could have left it as 0.17.0 and updated it later as well.